[docs] TLS routing behind l7 load balancer preview (#25424)
gravitational/teleport: docs/pages/management/operations/tls-routing.mdx

  opened={true}
  scope={["oss", "enterprise"]}
  scopeOnly={true}
  min="13.0"
>

>   TLS routing behind layer 7 (HTTP/HTTPS) load balancers and reverse proxies
>   support is available in Preview starting from Teleport `13.0`.

Reviewer: ```suggestion
Support for TLS routing behind layer 7 (HTTP/HTTPS) load balancers and reverse proxies is available in Preview starting from Teleport `13.0`.
```

</Details>

> Starting from version `8.0`, Teleport supports TLS routing. In this mode, all
> client connections are wrapped in TLS and multiplexed on one Teleport proxy
> port.

Reviewer: ```suggestion
In TLS routing mode, all Teleport client connections are wrapped in TLS and multiplexed on one Teleport Proxy Service port.
```
- v8 is long out of support, so I don't think we need to mention it
- Suggested to make this more concise and capitalize "Proxy Service"

Config version `v2` will prevent other listeners from being created unless
they are explicitly set in the proxy service configuration.

> ## Migrating from dual load balancer setups

Reviewer 2: The "dual load balancer setup" is quite vague. Additionally in TLS Routing mode we are forwarding by design all the traffic to the  web_proxy_addr so there is no any additional steps needed to be preformed between TLS Routing migration and L7 LB TLS Routing migration.
